docs/operator-guide.md

```diff
@@ -0,0 +1,191 @@
+# Atlas operator guide
+
+## Table of contents
+
+- [Basics](#basics)
+  - [Introduction](#introduction)
+  - [Prerequisites](#prerequisites)
+  - [Running Atlas](#running-atlas)
+  - [Configuring Atlas](#configuring-atlas)
+    - [App name](#app-name)
+    - [App logo](#app-logo)
+    - [App favicon](#app-favicon)
+    - [Categories](#categories)
+  - [Operating Atlas](#operating-atlas)
+    - [Content blocking](#content-blocking)
+    - [Kill switch](#kill-switch)
+- [Required services](#required-services)
+  - [Orion](#orion)
+  - [Query Node](#query-node)
+  - [RPC Node](#rpc-node)
+  - [Member faucet](#member-faucet)
+  - [Avatar service](#avatar-service)
+- [Optional services](#optional-services)
+  - [Social previews generation](#social-previews-generation)
+  - [Price feed](#price-feed)
+  - [Geolocation service](#geolocation-service)
+  - [Captcha](#captcha)
+- [Analytics](#analytics)
+  - [Sentry](#sentry)
+  - [Livesession](#livesession)
+  - [Usersnap](#usersnap)
+  - [Distribution logs](#distribution-logs)
+
+## Basics
+
+### Introduction
+
+This guide will help you get started with running your own instance of Atlas (a "gateway"). Atlas is a frontend single page application that allows users to interact with the Joystream network - basic functionalities include watching and publishing content, managing channels, issuing and trading NFTs, and more. For a brief introduction on how Atlas works and what services it interacts with, you may want to start with the [architecture overview](./architecture.md). To run a gateway, you will need to host a publicly available deployment of the frontend app, but you will also need to run a few other services that Atlas depends on. We will walk you through the process of setting up all of these services.
+
+**Tip:** Atlas provides built-in "admin tools" panel that can be useful when running your own instance. You can access it in the app by pressing Ctrl+Shift+D.
+
+### Prerequisites
+
+- Basic knowledge of how to use the command line, how to operate Docker, and how to host a web application
+- Domain name (e.g. `example.com`) to make your gateway publicly available
+- Server to deploy required services (e.g. VPS, dedicated server, or cloud instance)
+- Some kind of web server (e.g. Nginx) or PaaS (e.g. Vercel) to host the frontend app
+
+### Running Atlas
```

Comment:
Maybe I'm nitpicking here, but I'm not sure if this section should be first. By reading this article, I'm getting the impression, that you need to **first** run the atlas, and **then** configure it(without building it again). I know that this guide is aimed at rather to technical people, and most people will figure this out, but on the other hand, someone could not get it.  Maybe we should add at least just one sentence in `Configuring Atlas`, that you need to build Atlas after configuring the Atlas? WDYT?

Reply:
Good call, I will mention this

Reply:
Added in c867be77039f1015bee43840e54e29ed4dc20aa4